libs/server/Resp/ArrayCommands.cs

```diff
@@ -795,16 +806,64 @@ private bool NetworkTYPE<TGarnetApi>(int count, byte* ptr, ref TGarnetApi storag
             return true;
         }
 
-        private bool NetworkMODULE<TGarnetApi>(int count, byte* ptr, ref TGarnetApi storageApi)
+        private bool NetworkMODULE<TGarnetApi>(int count, byte* ptr, ref TGarnetApi storageApi, CustomCommandManager customCommandManager)
             where TGarnetApi : IGarnetApi
         {
-            if (count != 1)
+            if (count < 1) // At least one subcommand is required
                 return AbortWithWrongNumberOfArguments("MODULE", count);
 
-            // TODO: pending implementation for module support.
-            while (!RespWriteUtils.WriteError(CmdStrings.RESP_ERR_GENERIC_UNK_CMD, ref dcurr, dend))
-                SendAndReset();
+            // Read sub-command
+            if (!RespReadUtils.TrySliceWithLengthHeader(out var subCommand, ref ptr, recvBufferPtr + bytesRead))
+                return false;
 
+            if (subCommand.SequenceEqual(CmdStrings.LOAD) || subCommand.SequenceEqual(CmdStrings.load))
```

Comment:
Should we call this `MODULE LOAD` since existing clients might try to load non-compliant modules from other systems with this call? Should it instead be `MODULE LOADCS`?

Reply:
Changed the command to LOADCS.